CODEOWNERS

```
@@ -0,0 +1,13 @@
+# All helm chart related stuff should belong to the helm charts team
+* @adfinis-sygroup/helm-charts
+
+# Add specific owners to specific charts
+/charts/argoconfig    @adfinis-sygroup/helm-charts @hairmare
+/charts/back8sup      @adfinis-sygroup/helm-charts @tonit @hairmare
+/charts/barman        @adfinis-sygroup/helm-charts @hairmare @michaelimfeld
+/charts/caasperli     @adfinis-sygroup/helm-charts @hairmare @blaserd
+/charts/common        @adfinis-sygroup/helm-charts @hairmare
+/charts/kasperleyn    @adfinis-sygroup/helm-charts @hairmare @blaserd
```

```suggestion
/charts/caasperli     @adfinis-sygroup/helm-charts @hairmare
/charts/common        @adfinis-sygroup/helm-charts @hairmare
/charts/kasperleyn    @adfinis-sygroup/helm-charts @hairmare 
```
I think we should not add @ blaserd since he has finished his internship.

You can probably also remove me from the file since I'm in @adfinis-sygroup/helm-charts anyway.

---

Right, updated the file to reflect your suggestions